src/main/java/foxtrot/uniform/charlie/kilo/model/dto/OrderResponseDto.java

```diff
@@ -0,0 +1,50 @@
+package foxtrot.uniform.charlie.kilo.model.dto;
+
+import java.util.Arrays;
+
+public class OrderResponseDto {
+    private Long id;
+    private Long[] ticketIds;
```

Comment:
why array?

Reply:
I've fixed but @KseniiaMakarova can you confirm that this is a bad practice? Arrays has better performance than any Collections, right?

Reply:
@Artificial-Friend we should better use List, not array, in this case